### PR TITLE
Beginning story

### PR DESCRIPTION
**Pull request by Qiangong2**
Spurs the start of the story. Took me long enough :D

### CODE REVIEW

File: mods/tuxemon/maps/maple_house.tmx [hunk 1]
```diff
@@ -59,16 +61,36 @@
...
     <property name="act3" value="npc_face npc_wife,down"/>
-    <property name="cond1" value="is player_at"/>
-    <property name="cond2" value="is player_facing up"/>
+    <property name="cond1" value="is player_facing_tile"/>
     <property name="cond3" value="is button_pressed K_RETURN"/>
    </properties>
   </object>
+  <object id="40" name="create knight" type="event" x="192" y="96" width="16" height="16">
+   <properties>
+    <property name="act1" value="create_npc knight3,7,11,knight,stand"/>
```
Comment (xirsoi) [inline mods/tuxemon/maps/maple_house.tmx]: Remove the sprite name from this command, that functionality is deprecated and will generate warnings in the logs.
Don't delete any commas though, or the command will fail.

File: mods/tuxemon/maps/player_house_downstairs.tmx [hunk 2]
```diff
@@ -85,5 +85,20 @@
    </properties>
   </object>
   <object id="28" name="Player Spawn" type="event" x="80" y="64" width="16" height="16"/>
+  <object id="29" name="create mom" type="event" x="16" y="48" width="16" height="16">
+   <properties>
+    <property name="act1" value="create_npc npc_mom,9,4,girl1,stand"/>
```
Comment (xirsoi) [inline mods/tuxemon/maps/player_house_downstairs.tmx]: sprite name here too

File: mods/tuxemon/maps/taba_town.tmx [hunk 3]
```diff
@@ -375,11 +375,255 @@
     <property name="cond1" value="is variable_set battle_lost_faint:true"/>
    </properties>
   </object>
-  <object id="124" name="Tuxemart Sign" type="event" x="528" y="192" width="16" height="16">
+  <object id="125" name="bring on the nurses" type="event" x="352" y="640" width="16" height="16">
    <properties>
-    <property name="act1" value="dialog Tuxemart - A place to buy and sell items"/>
-    <property name="cond1" value="is player_facing_tile"/>
-    <property name="cond2" value="is button_pressed K_RETURN"/>
+    <property name="act1" value="create_npc nurse1,37,39,nurse,stand"/>
```
Comment (xirsoi) [inline mods/tuxemon/maps/taba_town.tmx]: more sprite names; i'll stop mentioning these, but please get rid of all of them.

**Comment by Qiangong2** [comment]
resolved @xirsoi

**Comment by Qiangong2** [comment]
Sorry about the last commit, I forgot that was still in there :P

**Comment by xirsoi** [comment]
I approve!

**Comment by bitcraft** [comment]
@xirsoi @Qiangong2 thank you for all your work over the weekend verifying this.

**Pull request merged**